### DISCUSSION TITLE
Manually set camera id

### DISCUSSION THREAD

**Discussion by Tom60chat** [natario1/CameraView#1179] [Ideas]
It will be nice to be able to manually set the camera ID, for multi-lens phones (ex: macro, ultra wide)

Sometings like :
```kotlin
cameraView.setCameraId("21")
```

**Comment by hardikpatelwsc** [comment]
I actually needed this functionality and that's why I was referring the discussions section and I found this, this is actually very important because most of the phones nowadays have multiple back cameras, and for that just FRONT & BACK options are not that much useful. I wanted to access ultrawide camera but right now there is no such option to use that.
